### PR TITLE
 #185218928 Users should be able to search 

### PR DESCRIPTION
**Pull request by irakevine**
## What does this PR do?
Users should be able to search 

## Description of Task to be completed?
- Given that a user is logged in when the user enters data into the search component, then the records displayed in the table should reflect all possible records relevant to the content of the data entered by the user in the search component

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout <ft-search-product-#185218928>`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?
[#185218928 ](https://www.pivotaltracker.com/story/show/185218928)

## Screenshots (if appropriate)

Uploading Screen Recording 2023-06-18 at 20.12.51.mov…

Uploading Screen Recording 2023-06-18 at 20.19.14.mov…

### CODE REVIEW

**Review by nishimweprince — APPROVED** [review]
Excellent work!!

**Review by bergad37 — APPROVED** [review]
Great work done!

**Review by Dyves1 — APPROVED** [review]

**Review by cyubahiro367 — APPROVED** [review]

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -154,4 +166,75 @@ const Navbar = () => {
...
+
+  const dispatch = useDispatch();
+
+  const [formData, setFormData] = useState({
+    name: 'Product',
+    categoryIds: 76,
+    price: 0,
+    size: 20,
+    page: 1
+  });
```
Comment (niyontwali) [inline src/components/Navbar.jsx]: Why are these data hardcoded?

**Review by niyontwali — CHANGES_REQUESTED** [review]

File: src/components/Navbar.jsx (hunk 1 above)
Comment (irakevine) [reply src/components/Navbar.jsx]: This is when the user clicks on the search button and there is no actual search.,
it's  data that comes by default when user click on search button

**Review by jcharles921 — APPROVED** [review]
LGTM

**Review by niyontwali — APPROVED** [review]

**Pull request merged**